### PR TITLE
PEAR-1736 changes Mutation Freq and OncoMatrix cards case counts logic

### PR DESCRIPTION
**Pull request by wteouchicago**
## Description
For Mutation Frequency and OncoMatrix, update the case counts displayed on the card so that they include cases with CNV data as well.

## Checklist

- [N/A] Added proper unit tests
- [N/A] ] Left proper TODO messages for any remaining tasks
- [N/A] ] Scanned for web accessibility with **aXe**, and mitigated or documented
      flagged issues

## Screenshots/Screen Recordings (if Appropriate)
A project like APOLLO-LUAD has CNV data but no SSM data. Note ProteinPaint says 0 while Mutation Frequency and OncoMatrix  show >0 cases.
Screenshot 2024-01-10 at 7 34 38 PM

TCGA-BRCA has both CNV and SSM.
Screenshot 2024-01-10 at 7 35 42 PM

FM-AD has no CNV data and, while I'm not logged in, has no SSM data available.
Screenshot 2024-01-10 at 7 44 05 PM

### CODE REVIEW

File: packages/core/src/features/cohort/tests/mockData.ts [hunk 1]
```diff
@@ -13,6 +13,7 @@ export const MOCK_COHORTS: Cohort[] = [
       genesCount: 24568,
       mutationCount: 22233445566,
       ssmCaseCount: 23450,
+      cnvOrSsmCaseCount: 28400,
```
Comment (wteouchicago) [inline packages/core/src/features/cohort/tests/mockData.ts:16]: @craigrbarnes I made these numbers up. Let me know if I should do something else.

**Comment by wteouchicago** [comment]
@craigrbarnes feel free to toss this. It started out as a POC and for investigating downstream effects.

File: packages/core/src/features/cohort/tests/mockData.ts (hunk 1 above)
Comment (craigrbarnes) [reply packages/core/src/features/cohort/tests/mockData.ts:16]: They look acceptable to me, any number > 0 should be fine.

File: packages/core/src/features/cohort/cohortCountsQuery.ts [hunk 2]
```diff
@@ -88,11 +92,16 @@ const CountsGraphQLQuery = `
           total
         }
       }
-     ssmsCases : cases {
+      ssmsCases : cases {
```
Comment (paribartandhakal) [inline packages/core/src/features/cohort/cohortCountsQuery.ts:95]: OCDing!

**Review by paribartandhakal — APPROVED** [review]
It looks good to me. Easy to follow 👍

File: packages/core/src/features/cohort/cohortCountsQuery.ts (hunk 2 above)
Comment (wteouchicago) [reply packages/core/src/features/cohort/cohortCountsQuery.ts:95]: it was staring at me...

**Review by craigrbarnes — APPROVED** [review]
Looks good and works as expected

**Pull request merged**